integrations/standalone/src/mock/validation-mock.ts

```diff
@@ -13,12 +13,12 @@ const validateDeep = (components: Array<ComponentData>, validations: Array<Valid
       validateDeep(c.config.components as Array<ComponentData>, validations);
     }
     if (c.type === 'Input') {
-      if (c.config.value === undefined || (c.config.value as string).length === 0) {
+      if (typeof c.config.value !== 'string' || c.config.value.length === 0) {
```

Comment:
For me more easy to read would be:
`(typeof c.config.value === 'string' && c.config.value.length === 0)`